src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Delivery/slip.html.twig

```diff
@@ -76,7 +74,7 @@
             <h3 class="card-header">
               <i class="material-icons">settings</i> {{ 'Delivery slip options'|trans }}
             </h3>
-            <div class="card-block row">
+            <div class="card-body">
```

Comment:
You didn't remove the `card-text` div here, is it on purpose? I though it was only supposed to contain text but this one contains HTML